README.md

```diff
@@ -59,28 +59,68 @@ the new one and cause confusion).
 
 If the `--host-port` flag is omitted, the emulator will choose a random port,
 which has a high likelihood of not being in use. The emulator will output which
-address to use by displaying a line like `export
-FIRESTORE_EMULATOR_HOST=::1:8195`.
+address to use by displaying a line like `export FIRESTORE_EMULATOR_HOST=::1:8195`.
 
 Note that, if the local IP address is an IPv6 address (like `::1`), then you
 will need to put square brackets around the address for compatibility with Go's
-parsing. In this example, that would look like `export
-FIRESTORE_EMULATOR_HOST=[::1]:8195`.
+parsing. In this example, that would look like `export FIRESTORE_EMULATOR_HOST=[::1]:8195`.
 
 ## Local Endpoints
 
 You can hit the endpoints with curl or Postman.
 
+Try:
+
+```
+curl --request GET 'http://localhost:8080/challenge' \
+     --header 'X-Forwarded-Proto: https'
+```
+
+### HTTPS
+
+The local dev server uses http so you must send a fake https header to prevent the request
```

Comment:
nit: s/http/HTTP and s/https/HTTPS
nit: s/prevent the request/prevent the request from